http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeV2ServiceInterp.scala

```diff
@@ -554,12 +594,13 @@ class RuntimeV2ServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
               case None => List.empty
             }
           } yield
-          // runtimesUserIsCreator: runtimes user is creator
+          // runtimesUserIsCreatorWithoutWorkspaceId: runtime user is creator, legacy V1 without workspace
```

Comment:
The logic above feels a bit dicey to me, would appreciate some extra scrutiny :)